src/Angor/Shared/Services/NetworkService.cs

```diff
@@ -100,7 +100,10 @@ public async Task CheckServices(bool force = false)
                     try
                     {
                         var uri = new Uri(indexerUrl.Url);
-                        var response = await _httpClient.GetAsync($"{uri}api/stats/heartbeat");
+                        
+                        var blockUrl = Path.Combine(uri.AbsoluteUri, "api", "v1", "block-height", "0");
```

Comment:
This is very different way to check status, it will just return genesis, or is it only to test the api is available?